.github/workflows/detect-breaking-changes-build.yml

```diff
@@ -43,18 +43,18 @@ jobs:
         name: buildPr
         path: './pr/pr_built_packages.zip'
 
-  buildMain:
-    name: Build Main
+  buildBase:
+    name: Build Base
     runs-on: ubuntu-latest
     defaults:
       run:
-        working-directory: './main'
+        working-directory: './base'
 
     steps:
     - uses: actions/checkout@v2
       with: 
-        path: './main'
-        ref: 'main'
+        path: './base'
+        ref: ${{ github.event.pull_request.base.ref }}
```

Comment:
Is there any security concern around this?

Reply:
Wasn't sure if we expose any secrets in this action. If not, should be fine

Reply:
Luckily this action uses the `pull_request` trigger which shouldn't have access to the secrets, so I think we should be safe, too. 👍